splunk/src/test/java/com/splunk/SDKTestCase.java

```diff
@@ -92,9 +92,23 @@ public static Integer getJavaVersion() {
     }
 
     @BeforeClass
-    public static void preClassLoadActions() {
-        // Bypass the certification validation here.
+    public static void preClassLoadActions() throws IOException {
+        // To Bypass the certification validation.
         HttpService.setValidateCertificates(false);
```

Comment:
Could we have separate tests:
1. For simulating the failure of an SSL cert validation 
2. and/or when setting to `true`, the ability to connect to the HTTPS resource with a valid SSL certificate?